lib/stdlib/doc/src/math.xml

```diff
@@ -102,9 +102,20 @@ erf(X) = 2/sqrt(pi)*integral from 0 to X of exp(-t*t) dt.</pre>
 
     <func>
       <name name="pi" arity="0" since=""/>
-      <fsummary>A useful number.</fsummary>
+      <fsummary>Ratio between circumfence and diameter of the circle.</fsummary>
       <desc>
-        <p>A useful number.</p>
+          <p>Ratio between circumfence and diameter of the circle.</p>
+          <p>Floating point approximation of mathematical constant pi.</p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="tau" arity="0" since=""/>
+      <fsummary>Ratio between circumfence and radius of the circle aka full turn.</fsummary>
```

Comment:
```suggestion
      <fsummary>Ratio of circumfence to radius of a circle.</fsummary>
```
Do not use abbreviations like "aka" in the documentation. Actually, I would leave that part out entirely here, it is not necessary in the summary.